### ISSUE TITLE
Instructions for getting the demo to run locally

### ISSUE THREAD

**Issue by 1fifoto** [ABASystems/react-interactive-tutorials#5]
I'd like to use your package. To do that I've installed your package on my machine, and I am trying to get the demo-site to run locally. However, I've been unsuccessful so far. It seems that I'm unclear about your dependency with uptick-demo-site. For example, what command or commands do I run to get them to play nicely together. Can you give me some suggestions or pointers.

**Comment by 1fifoto** [comment]
Here is what I did. Does this make sense? What are your thoughts? Would it be possible to add this to the README.md file?
* Read in uptick-demo-site about using Jekyll.
* Followed Jekyll Installation instructions at https://jekyllrb.com/docs/installation/. I'm on MacOS.
  * Updated Apple's Xcode: xcode-select --install
  * Verified ruby version > 2.2.5: ruby -v
  * Installed bundler and jekyll gems: sudo gem install bundler jekyll
* In demo-site root directory created Gemfile containing
  * source 'https://rubygems.org'
  * gem 'jekyll'
* Executed "jekyll serve" command in the context of the bundle: bundle exec jekyll serve
  * For me to get this to work I needed to create a Gemfile in the demo-site directory containing the lines:
    * source 'https://rubygems.org'
    * gem 'jekyll'
* In browser went to: http://127.0.0.1:4000
* TADA!
![screen shot 2018-09-14 at 10 59 01 pm](https://user-images.githubusercontent.com/1581602/45582230-d5a55e80-b871-11e8-939b-c796032ce65e.png)

**Comment by 1fifoto** [comment]
Just as a final note for developers. 

Obviously if you make modifications to the react-interactive-tutorial then you will need to change directory to the root directory (cd git/react-interactive-tutorial) and run an npm build (npm run build). 

To have the demo-site directly access your modification I changed the sample-tutorial,js file from

    import interactiveTutorials from 'react-interactive-tutorials'
    import {
    ...
    } from 'react-interactive-tutorials'

to

    import interactiveTutorials from '../dist/react-interactive-tutorials'
    import {
    ...
    } from '../dist/react-interactive-tutorials'

Then from the demo-site directory (cd git/react-interactive-tutorial/demo-site) and run an npm build (npm run build). After you do this and if "jekyll serve" is running then it senses that the demo-site has been changed and the demo-site is reloaded.

I'm sure there are other ways to do this, but this worked for me.
